testChanges.sh

```diff
@@ -0,0 +1,7 @@
+cargo build --locked --no-default-features --all
```

Comment:
Please move to [`contrib/`](https://github.com/romanz/electrs/tree/master/contrib) and the below:

```suggestion
#!/bin/bash
cd `dirname $0`/..
cargo build --locked --no-default-features --all
```

Reply:
Please move to `contrib/`

Reply:
Will update this in the latest PR.